test/sanity/ignore.txt

```diff
@@ -983,6 +983,8 @@ lib/ansible/modules/cloud/openstack/os_project_access.py validate-modules:doc-mi
 lib/ansible/modules/cloud/openstack/os_project_access.py validate-modules:parameter-type-not-in-doc
 lib/ansible/modules/cloud/openstack/os_project_info.py validate-modules:doc-missing-type
 lib/ansible/modules/cloud/openstack/os_project_info.py validate-modules:parameter-type-not-in-doc
+lib/ansible/modules/cloud/openstack/os_qos_policy.py validate-modules:doc-missing-type
+lib/ansible/modules/cloud/openstack/os_qos_policy.py validate-modules:undocumented-parameter
```

Comment:
You accept 'availability_zone' as a parameter (it comes from openstack_full_argument_spec) so it should be documented.

I'm surprised it's not included in the fragment ( ./lib/ansible/plugins/doc_fragments/openstack.py ).  But this certainly shouldn't be grounds for an entry in ignore.txt